recipes/configs/generate.yaml

```diff
@@ -0,0 +1,32 @@
+
+# Model arguments
+model:
+  _component_: torchtune.models.llama2.llama2_13b
```

Comment:
Should we call it generate_13b?

Reply:
Not really - the model itself doesn't change during inference. The inference script is responsible for setting up the caches and the calling ```eval()``` on the model to disable all of the stochastic operations.